vault/resource_database_secret_backend_connection.go

```diff
@@ -1068,6 +1109,19 @@ func getMySQLConnectionDetailsFromResponse(d *schema.ResourceData, prefix string
 			result["tls_ca"] = v.(string)
 		}
 	}
+
+	// cloud specific
+	if v, ok := data["auth_type"]; ok {
```

Comment:
could we also add vault version checking for these? Thanks!

Reply:
is that redundant here? if vault doesn't support it, there won't be any in the response and it'll just be blank?